internal/scheduler/scheduling_algo_test.go

```diff
@@ -338,26 +338,6 @@ func TestSchedule(t *testing.T) {
 			},
 			expectedScheduledIndices: []int{0},
 		},
-		"UnifiedSchedulingByPool": {
```

Comment:
I think we want to keep these tests.

Reply:
I've put back `Schedule gang job over multiple executors` but I think `UnifiedSchedulingByPool` is now the same as `scheduling`